### PR TITLE
Add try catch block for loading locale files

### PR DESCRIPTION
**Pull request by samuveth**
Changes proposed in this pull request:
- Add try catch block for loading locale files
- Show select language if locale saved in local storage no longer exists

### CODE REVIEW

File: src/components/Modal/About.vue [hunk 1]
```diff
@@ -26,7 +26,9 @@
     <div class="m-4 p-4 mt-3 border rounded-2 text-white">
       <div class="d-flex">
         <span v-text="$t('language')" class="flex-auto text-gray mr-1" />
-        <a @click="changeLang()">{{ languages[$i18n.locale].name }}</a>
+        <a @click="changeLang()">{{
+          languages[$i18n.locale]?.name || $t('selectLanguage')
```
Comment (bonustrack) [inline src/components/Modal/About.vue:30]: We should show the current locale used here, not "Select language"

**Comment by bonustrack** [comment]
@samuveth Then this should be fixed at the source, not here. We should make sure a correct language is used all the time. If someone have a wrong locale on localStorage we should default to English.

**Comment by samuveth** [comment]
> @samuveth Then this should be fixed at the source, not here. We should make sure a correct language is used all the time. If someone have a wrong locale on localStorage we should default to English.

@bonustrack I think I came of with something that should work like that.

**Comment by samuveth** [comment]
@bonustrack after failing the to load the translation file, it fails to find the keys and falls back to english.

Screen Shot 2021-04-17 at 9 09 02 PM

**Pull request merged**